style.css

```diff
@@ -1 +1,267 @@
-/*  Add your styling here */
+/* General rules */
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    font-style: italic;
+    font-family: 'Times New Roman', Times, serif;
+}
+
+/* Header */
+.header {
+    width: 100%;
+    height: 25vh;
+}
+
+.header-container {
+    display: flex;
+    justify-content: space-between;
+}
+
+.header-img img {
+    /* width: 25vw; */
+    height: 15vh;
+    margin-top: 1rem;
+    margin-left: 1rem;
+    cursor: pointer;
+}
+
+.menu-btn {
+    padding-right: 2rem;
+    padding-top: 1rem;
+    font-size: 2rem;
+    cursor: pointer;
+}
+
+.menu-btn:hover {
+    font-size: 2.5rem;
+    color: rgb(190, 110, 35);
+    transition: all 0.5s ease;
+}
+
+/* Navigation Menu */
+
+.navbar {
+    display: none;
+}
+
+.navbar-list {
+    display: flex;
+    list-style-type: none;
+    justify-content: flex-end;
+    padding-right: 5rem;
+    font-size: 1rem;
+}
+
+.navbar-list a {
+    cursor: pointer;
+    text-decoration: none;
+    color: rgb(190, 110, 35);
+    margin-left: 3rem;
+}
+
+.navbar-list a:hover {
+    font-size: 1.1rem;
+    transition: all 0.4s ease;
+}
+
+.navbar-list a {
+    padding: 0.5rem;
+    background-color: rgb(190, 110, 35);
+    color: #f2f2f2;
+    border-radius: 5px;
+    text-transform: uppercase;
+}
+
+/* Main */
+main {
+    margin: auto 0.5rem;
+}
+
+/* Main first section */
+.header-desc {
+    float: right;
+    width: 50%; 
+    font-size: 1rem;   
+}
+
+.hero-desc {
+    margin: 1rem;
+    text-align: justify;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-around;
+    margin: 0 2rem 1rem;
+}
+
+.hero h1 {
+    margin-top: 1rem;
+    padding-top: 0.5rem;
+    text-align: center;
+    margin-bottom: 0.5rem;
+    border-top: 1px solid rgb(190, 110, 35);
+    width: 100%;
+}
+
+.hero-desc p {
+    padding: auto 1rem;
+}
+
+.hero img {
+    width: 100%;
+    height: 50vh;
+    margin-bottom: 1rem;
+}
+
+/* Main second section */
+.cakes-pictures-container {
+    display: flex;
+    flex-wrap: wrap;
+    width: 100%;
+    margin-bottom: 2rem;
+}
+
+.cakes-pictures-container a {
+    width: 50%;
+    cursor: pointer;
+}
+
+.cakes-pictures {
+    width: 100%;
+    height: 20vh; 
+    padding: 0.3rem;
+}
+
+/* Footer */
+.footer-list {
+    width: 100%;
+    padding-left: 2rem;
+    border-top: 1px solid rgb(190, 110, 35);
+    padding-top: 1rem;
+    margin-bottom: 1rem;
+}
+
+.footer-list li {
+    list-style-type: none;
+    cursor: pointer;
+}
+
+.footer-list p {
+    font-weight: 600;
+}
+
+.footer-list-basics, 
+.footer-list-GetInTouch, 
+.footer-list-policies {
+    margin-bottom: 1rem;
+}
+
+.company-name {
+    text-align: center;
+    margin-bottom: 1rem;
+}
+
+/* Responsive Rules */
+
+@media (min-width: 600px) and (max-width: 900px) {
```

Comment:
`max-width` is not needed here.
Check `readme.md` for more detail on how to create mobile first styles
Try changing this to `@media (min-width: 600px)` and see what happens 😉

Reply:
Yes, you are right, it was unnecessary, deleted the max-width.

`@media (min-width: 600px) `